extensions/chromium/preferences_schema.json

```diff
@@ -220,6 +220,11 @@
       "description": "The color is a string as defined in CSS. Its goal is to help improve readability in high contrast mode",
       "type": "string",
       "default": "CanvasText"
+    },
+    "enableAutoLinking": {
+      "description": "Automatically detect URLs in the text and create links for them",
```

Comment:
This description should perhaps match the JSDocs for the `PDFViewer`/`PDFPageView` parameter better?